script.js

```diff
@@ -64,59 +64,91 @@ function resetGame() {
     pumpkinMove.pause();
     reaperMove.cancel();
     grimReaperZone.removeEventListener('mouseleave', moveReaper);
+
 }
 
 // code here to activate win modal displaying
 function win() {
     winModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
```

Comment:
These two lines are the same for `win()` and `lose()` - put them in `resetGame()`?